src/v2/providers/tasks.ts

```diff
@@ -33,21 +33,68 @@ import {
 import {
   AuthData,
   onDispatchHandler,
-  RateLimits,
   Request,
-  RetryConfig,
 } from '../../common/providers/tasks';
 import * as options from '../options';
 import { HttpsFunction } from './https';
+import { Expression, Field } from '../expressions';
 
-export { AuthData, RateLimits, Request, RetryConfig };
+export { AuthData, Request };
+
+/** How congestion control should be applied to the function. */
+export interface ParameterizedRateLimits {
```

Comment:
Why do we do this instead of wrapping RetryConfig/RateLimits in `../../common/providers/tasks` instead?

Is params only supported in v2?

Reply:
I'm in favor of letting v2 functions be configured with Params as well.